### DISCUSSION TITLE
I think the recent v1.4.2 release note is wrong.

### DISCUSSION THREAD

**Discussion by DahoonMoon** [MystenLabs/sui#12912] [General]
Release note says update to protocol version 14, but isn't it actually update to version 15?

== release note ==
[mainnet-v1.4.2](https://github.com/MystenLabs/sui/releases/tag/mainnet-v1.4.2)
https://github.com/MystenLabs/sui/pull/12548 This release updates the protocol version to 14.
==============

**Comment by ekfbdjskmsnhsjso** [comment]
damn so cute
